numpy_challenge.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+if __name__ == "__main__":  # creatint 3 different numpy arrays
+    arr_1 = np.eye(3, dtype=int)
+    arr_2 = np.arange(9).reshape((3, 3))
+    arr_3 = np.fromstring('7 3', dtype=int, sep=' ')  # fav number of Sheldon Cooper
+
+
+def matrix_multiplication(a, b):  # a and b are matrices
+    return np.matmul(a, b)
+
+
+def multiplication_check(a):  # a is an array of matrices
+    for i in range(1, len(a)):
+        if np.shape(a[i])[0] != np.shape(a[i - 1])[1] or np.shape(a[i])[1] != np.shape(a[i - 1])[0]:
```

Comment:
1. Нейминг переменных!!! Не понятно, что такое `a`
2. Практически у вех функций для работы с эрреями в numpy есть соответствующие методы и атрибуты. В коде они намного лучше читаются
```suggestion
def multiplication_check(matrices_list):  # a is an array of matrices
    for i in range(1, len(matrices_list)):
        if matrices_list[i].shape[0] != matrices_list[i - 1].shape[1]:
```